src/mutations/article/publishArticle.ts

```diff
@@ -50,6 +57,21 @@ const resolver: GQLMutationResolvers['publishArticle'] = async (
     throw new UserInputError('content is required')
   }
 
+  const fieldName = 'publishArticle'
+  const pass = await checkOperationLimit({
+    user: viewer.id || viewer.ip,
```

Comment:
viewer.ip will never be used as only logged-in user can publish articles by now?

Reply:
And as opercation limit is checked here,  rateLimitDirective (@rateLimit) on `publishArticle` in schema need to be removed